services/director/setup.py

```diff
@@ -3,24 +3,24 @@
 
 INSTALL_REQUIRES = [
 'docker==3.5.0',
-'Flask==1.0.1',
 'tenacity==4.12.0',
+'aiohttp==3.3.2',
 'aiohttp_apiset==0.9.3',
 'requests==2.19.1',
 ]
 
 setup(
-    name='director-service',
+    name='simcore-director-webserver',
     version="0.1.0",
-    description='oSparc Director service',
+    description='oSparc Director webserver service',
```

Comment:
This is not the naming we agreed, right? it would be following the folder structure (in singular),i.e. 
- package name ``simcore-service-director``
- import name ``simcore_service_director`` (we could have an alias as ``scs_director``)
- main entry point ``simcore-service-director``